src/app/components/ParaTimePicker/index.tsx

```diff
@@ -0,0 +1,119 @@
+  </Drawer>
+)
+
+type ParaTimePickerContentProps = Omit<ParaTimePickerProps, 'open'>
+
+const ParaTimePickerContent: FC<ParaTimePickerContentProps> = ({ onClose, onConfirm }) => {
+  const { t } = useTranslation()
+  const network = useSafeNetworkParam()
+  const layer = useLayerParam()
+  const [showNetworkMenu, setShowNetworkMenu] = useState(false)
+  const [selectedLayer, setSelectedLayer] = useState<undefined | Layer>()
+  const [selectedNetwork, setSelectedNetwork] = useState<undefined | Network>(Network.mainnet)
+  const [hoveredLayer, setHoveredLayer] = useState<undefined | Layer>()
+  const [hoveredNetwork, setHoveredNetwork] = useState<undefined | Network>()
+
+  return (
+    <Box>
+      <Box sx={{ mb: 5, color: 'red' }}>
+        <Logotype color={COLORS.brandExtraDark} />
+      </Box>
+      <IconButton
+        aria-label={t('paraTimePicker.toggleNetworkMenu')}
+        onClick={() => setShowNetworkMenu(!showNetworkMenu)}
+        sx={{
+          color: COLORS.brandDark,
+          ml: 3,
+        }}
+      >
+        {showNetworkMenu ? <KeyboardDoubleArrowLeftIcon /> : <KeyboardDoubleArrowRightIcon />}
+      </IconButton>
+      <Divider />
+      <Box
+        onMouseLeave={() => {
+          setHoveredNetwork(undefined)
+        }}
+      >
+        <Grid container>
+          {!showNetworkMenu && (
+            <Grid xs={1} sx={{ maxWidth: '40px' }}>
+              <NetworkMenuIcon />
+            </Grid>
+          )}
+          {showNetworkMenu && (
+            <Grid xs={3}>
+              <NetworkMenu
+                activeNetwork={network}
+                hoveredNetwork={hoveredNetwork}
+                selectedNetwork={selectedNetwork}
+                setHoveredNetwork={setHoveredNetwork}
+                setSelectedNetwork={setSelectedNetwork}
+              />
+            </Grid>
+          )}
+          <Grid xs={3}>
+            <ParaTimeMenu
+              activeLayer={layer}
+              hoveredLayer={hoveredLayer}
+              selectedLayer={selectedLayer}
+              setHoveredLayer={setHoveredLayer}
+              setSelectedLayer={setSelectedLayer}
+            />
+          </Grid>
+          <Grid xs={6}>
+            <ParaTimeDetails
+              activeLayer={layer}
+              hoveredLayer={hoveredLayer}
+              selectedLayer={selectedLayer}
+              network={selectedNetwork || network}
+            />
+          </Grid>
+        </Grid>
+
+        <Box sx={{ display: 'flex', justifyContent: 'flex-end', gap: 4 }}>
+          <Button onClick={onClose} color="primary" variant="outlined" sx={{ textTransform: 'capitalize' }}>
+            {t('common.cancel')}
+          </Button>
+          <Button
+            onClick={() => onConfirm(selectedNetwork!, selectedLayer!)}
+            disabled={!selectedLayer || !selectedNetwork}
+            color="primary"
+            variant="contained"
+          >
+            {t('common.select')}
+          </Button>
+        </Box>
```

Comment:
(smaller screen layout issue: buttons sometimes overlay the text:
![image](https://github.com/oasisprotocol/explorer/assets/3758846/bea9390a-0a30-487d-926f-33bbd204ece4)